homeassistant/components/camera/__init__.py

```diff
@@ -500,3 +547,31 @@ def _write_image(to_file, image_data):
             _write_image, snapshot_file, image)
     except OSError as err:
         _LOGGER.error("Can't write image to file: %s", err)
+
+
+async def async_handle_play_stream_service(camera, service):
+    """Handle play stream services calls."""
+    from homeassistant.components.media_player.const import (
+        ATTR_MEDIA_CONTENT_ID, ATTR_MEDIA_CONTENT_TYPE,
+        SERVICE_PLAY_MEDIA, DOMAIN as DOMAIN_MP)
+    from homeassistant.components.stream import request_stream
```

Comment:
We can move this import to the top.

Reply:
So you are OK making camera dependent on stream (and therefore pyAV)?

Reply:
I didn't move this yet, because it broke my tests trying to patch `request_stream`

Reply:
It's fine to import files and functions. You already import OUTPUT_FORMATS etc.

Update your patch to be `patch('homeassistant.components.camera.request_stream')`